packages/p2p-v2/src/components/PaymentMethodForm/PaymentMethodFormInput/PaymentMethodFormInput.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { THooks, TSelectedPaymentMethod } from 'types';
+import { Dropdown } from '@/components';
+import { TFormState } from '@/reducers/types';
+import { Button, Input, Text } from '@deriv-com/ui';
+import CloseCircle from '../../../public/ic-close-circle.svg';
+
+type TPaymentMethodFormInputProps = {
+    actionType: TFormState['actionType'];
+    availablePaymentMethods?: THooks.PaymentMethods.Get;
+    availablePaymentMethodsList: { text: string; value: string }[];
+    onAdd: (selectedPaymentMethod?: TSelectedPaymentMethod) => void;
+    reset: () => void;
+    selectedPaymentMethod: TFormState['selectedPaymentMethod'];
+};
+
+const PaymentMethodFormInput = ({
```

Comment:
@ameerul-deriv, do you really think this component is an `Input`? 🤔 
I wanted to separate it out, but couldn't come up with a good name for it. It couldn't fit into the `PaymentMethodField` component, so I decided to leave it there. Anyway, the best I could think of is creating two components out of it .i.e. `InputWithIcon` or something like that, and `DropdownWith? ClickableText? Button?`. But I don't think it's a form input.
 
```suggestion
const PaymentMethodFormInput = ({
```

Reply:
true its both a input and dropdown, its more of payment method selector since we're choosing which payment method we want to select and add to our payment methods. but regardless i think this should be one component because its rendering the Input/Dropdown that needs to be shown when adding/editing the ad.

So either we can go with `PaymentMethodSelector` or `PaymentMethodInputDropdown`

Reply:
what do you think

Reply:
I think `Autocomplete` would be a good name, since it functions like an autocomplete form

Reply:
niceee, yeah i'll write it as PaymentMethodFormAutocomplete

Reply:
Yeah, if a payment method is selected, then the `dropdown` is not rendered, the `input` with the `close` icon is rendered. If the payment method is not yet selected, then the dropdown is rendered. That's why I suggested separating them, because both never show up at once. You can choose one of the 2 names you and @adrienne-deriv agree on and move the card forward 👍.